### PR TITLE
#6: add --qif cli option

### PR DESCRIPTION
**Pull request by tomquirk**
see what you think @ArtS. Verryyy simple CLI arg implementation. Probably fine for now and it wouldn't be hard to extend slightly if needed.

### CODE REVIEW

**Comment by ArtS** [comment]
Hey @tomquirk , QIF is kind of the default operating mode, any reason for turning that into a parameter?

**Comment by tomquirk** [comment]
For myself at least, I don't need the QIF features, only the sqlite stuff. Basically I'd prefer to not have QIF files created in the first place instead of deleting them manually.

An easy compromise would be to have it export QIF by default and add an arg `--qif=false` to turn it off

**Comment by ArtS** [comment]
Totally see the point & the use case, I suppose I am just cautious not to break existing behaviour. Would adding `--qif=false` or `--no-qif` work for you?

**Comment by tomquirk** [comment]
yea will do 😄

**Comment by tomquirk** [comment]
@ArtS there ya go

File: README.md [hunk 1]
```diff
@@ -4,8 +4,7 @@
 
 I hated having to login to NAB's clunky Internet banking and fight with its forms to get my data, so I decided I'd better automate this process.
 
-This will export all your transactions for all your NAB accounts into nice QIF
-files. You can use QIF to load in your desktop or online account software, such as Quicken/MS Money/Mint/Xero etc.
+nab-export exports all of your transactions for all your NAB accounts. Using the `--qif` option, you can load exported QIF files into your desktop or online accounting software, such as Quicken/MS Money/Mint/Xero etc.
```
Comment (parxier) [inline README.md]: Wording needs to be changed to incorporate `--no-qif` switch now.

**Review by parxier — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ArtS) [reply README.md]: Good point, thanks браж

Comment (parxier) [reply README.md]: тамбовский :wolf: тебе браж

**Comment by tomquirk** [comment]
@parxier thanks for that 😄 @ArtS

**Review by ArtS — APPROVED** [review]

**Pull request merged**